ui/src/style/chronograf.scss

```diff
@@ -42,3 +41,4 @@
 @import 'src/logs/components/logs_table/LogsTable';
 @import 'src/logs/components/expandable_message/ExpandableMessage';
 @import 'src/logs/components/logs_message/LogsMessage';
+@import '../../node_modules/@influxdata/react-custom-scrollbars/dist/styles.css';
```

Comment:
Do we want to move node_module styles to a separate section?

Reply:
Ahh that's a good call